### PR TITLE
New package: python-colorama-0.3.3

### PR DESCRIPTION
**Pull request by okapia**
Another (transitive) dependency needed for vdirsyncer.

### CODE REVIEW

File: srcpkgs/python-colorama/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'python-colorama'
+pkgname=python-colorama
+version=0.3.3
+revision=1
+noarch=yes
+wrksrc=colorama-${version}
+build_style="python-module"
+python_versions="2.7 3.4"
+build_style="python-module"
```
Comment (asergi) [inline srcpkgs/python-colorama/template]: Duplicate `build_style="python-module"`

File: srcpkgs/python-colorama/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+hostmakedepends="python-setuptools python3.4-setuptools"
```
Comment (asergi) [inline srcpkgs/python-colorama/template]: `depends="python"`
`pycompile_module="colorama"`

File: srcpkgs/python-colorama/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+distfiles="${PYPI_SITE}/c/colorama/colorama-${version}.tar.gz"
+checksum=eb21f2ba718fbf357afdfdf6f641ab393901c7ca8d9f37edd0bee4806ffa269c
+
+post_install() {
+	vlicense LICENSE.txt LICENSE
+}
+
+python3.4-colorama_package() {
+	noarch=yes
+	pycompile_version="3.4"
```
Comment (asergi) [inline srcpkgs/python-colorama/template]: `pycompile_module="colorama"`
`depends="python3.4"`

File: srcpkgs/python-colorama/template [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+revision=1
+noarch=yes
+wrksrc=colorama-${version}
+build_style="python-module"
+python_versions="2.7 3.4"
+build_style="python-module"
+hostmakedepends="python-setuptools python3.4-setuptools"
+short_desc="Cross-platform colored terminal text in Python2"
+maintainer="Oliver Kiddle <okiddle@yahoo.co.uk>"
+license="BSD"
```
Comment (asergi) [inline srcpkgs/python-colorama/template]: `license="3-clause-BSD"`

**Comment by okapia** [comment]
Thanks for the review comments. Updated template pushed.

**Pull request merged**